pom.xml

```diff
@@ -401,7 +408,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>3.2.4</version>
+                <version>3.3.1-SNAPSHOT</version>
```

Comment:
This needs to be changed to 3.3.0

Reply:
Does it *have* to be 3.3.0? This AFAIK has been working without issue and while I don't like running snapshot dependencies, if we make this change it'll require a whole new release candidate build...

Reply:
This change does not affect the shading whatsoever. This does not require a new RC.

Reply:
Doesn't matter, you're changing the build... That's the way the release process has been setup to prevent risk being introduced for no sensible reason. Changes made after a RC Cut will require a new RC to be cut regardless of the change.